src/command.js

```diff
@@ -11,10 +11,9 @@ const list = val => val.split(',');
 export default () => {
   program
     .version('0.1.0')
-    .usage('[options]')
-    .option('-f, --from <n>', 'From block', parseInt)
-    .option('-t, --to <n>', 'To block', parseInt)
-    .option('-a, --addresses <n>', 'List of address', list);
+    .option('-a, --addresses <n>', 'List of address', list)
+    .option('-f, --from [n]', 'From block', -1)
```

Comment:
I would suggest using wording like `"default"` maybe or something it would be a better indicator than `-1` or we make a constant for example `defaultMode`

Reply:
I liked the constant Idea